charts/sdc/values.yaml

```diff
@@ -85,16 +97,29 @@ agents:
     ConnectionStrings__AgentsDatabase: "Host=postgresql; Database=agentsdb; Username=agentsadmin; Password=xxxxx; SearchPath=agents"
```

Comment:
This should be like

  database:
    name: agentsdb
    schema: agents
    host: postgresql
    port: 5432
    auth:
      user: agentsadmin
      password: "xxx"